lambdas/postgres-migration-count-tool/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "postgres_migration_count_tool_function_arn" {
+  value = aws_lambda_function.postgres_migration_count_tool.arn
+}
+
+output "postgres_migration_count_tool_role_arn" {
```

Comment:
This role arn is needed if we're using this for an ECS task.   It can be removed if we decide to avoid that.